aws/rust-runtime/aws-inlineable/src/http_request_checksum.rs

```diff
@@ -0,0 +1,303 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#![allow(dead_code)]
```

Comment:
Is this still needed? I'm guessing it's for the `pub(crate)` stuff but I wanted to be sure.

Reply:
It's mostly to keep CI on the `rust-runtime` crates happy.